labelbox/schema/export_params.py

```diff
@@ -34,3 +36,18 @@ class CatalogExportParams(DataRowParams):
 class ModelRunExportParams(DataRowParams):
     # TODO: Add model run fields
     pass
+
+
+def validate_array_length(array, max_length, array_name):
```

Comment:
Probably good to prefix this with `_` since it's a private function
`_validate_array_length`